source/agora/node/Ledger.d

```diff
@@ -228,6 +228,9 @@ public class Ledger
         this.enroll_man.clearExpiredValidators(block.header.height);
 
         foreach (idx, ref enrollment; block.header.enrollments)
+        {
+            this.enroll_man.pool.remove(enrollment.utxo_key);
+
             if (auto r = this.enroll_man.addValidator(enrollment,
                 block.header.height, this.utxo_set.getUTXOFinder()))
```

Comment:
this.enroll_man.pool.remove
The `remove` is run twice here.

Reply:
Where do you see it?

Reply:
https://github.com/bpfkorea/agora/pull/845#discussion_r435748483